boards/arm/nrf9160_innblue21/Kconfig.defconfig

```diff
@@ -42,4 +42,7 @@ endif # BOARD_NRF9160_INNBLUE21_NS
 config BT_HCI_VS
 	default y if BT
 
+config REGULATOR
+	default y if SENSOR
```

Comment:
Just curious, why only `if SENSOR`?

Reply:
according to the comments in the source file the switches power sensors